uint64 timestamp
bool block_release
string agent_name
string storage_name
string item_name
uint32 item_amount
